src/test/java/uk/nhs/digital/nhsconnect/lab/results/inbound/fhir/EdifactToFhirServiceTest.java

```diff
@@ -47,7 +51,7 @@ void testEdifactIsMappedToFhirBundle() {
 
         assertThat(bundle).isNotNull();
         assertThat(bundle.getEntry())
-            .hasSize(1)
+            .hasSize(2)
             .first()
```

Comment:
We're only extracting the first bundle entry and checking the contents. It may be worth also checking the second is as expected?

Reply:
Changed test in 9d42e9f to check the bundle size and all contents